src/ai/backend/agent/server.py

```diff
@@ -812,15 +821,28 @@ async def server_main(
 @click.option(
     "--debug",
     is_flag=True,
-    help="Enable the debug mode and override the global log level to DEBUG.",
+    help="This option will soon change to --log-level TEXT option.",
+)
+@click.option(
+    "--log-level",
+    type=click.Choice(LogLevel, case_sensitive=False),
+    default="info",
```

Comment:
default value of `--log-level` is not string type. it is `LogLevel` type.
```suggestion
    default=LogLevel.INFO,
```